tophatmonocle/yvm
# Yvm remove

## Dev prep
- npm install
- make install

## QA Steps
- install a version of yarn using `yvm install <version>`
- remove the version using `yvm remove <version>` and ensure that `~/.yvm/versions/v<version>` does not exist anymore
